docs/cli/reference.md

```
@@ -990,6 +993,31 @@ Generate a fresh anonymous install id (only while opted in)
 
 
 
+## `aoe mcp`
+
+Inspect the effective MCP server set (provenance, conflicts, drift)
+
+**Usage:** `aoe mcp <COMMAND>`
+
+###### **Subcommands:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)